contracts/src/LightClient.sol

```diff
@@ -291,7 +291,7 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
         IPlonkVerifier.VerifyingKey memory vk = VkLib.getVk();
 
         // Prepare the public input
-        uint256[8] memory publicInput;
+        uint256[7] memory publicInput;
```

Comment:
you didn't push more items to the public input, right now there is just 4 items.

reminder for us to edit this later.

Reply:
fixed in [59243df](https://github.com/EspressoSystems/espresso-sequencer/pull/1986/commits/59243df5edec5d83aff09fe3d9ae35b95dad3dcf)